adapter_pipelines/submit_example.json

```diff
@@ -0,0 +1,9 @@
+{
+  "WrapperSs2RsemSingleSample.format_map": "../../hca-dcp/format_map_example.json",
+  "WrapperSs2RsemSingleSample.submit_url": "http://api.ingest.staging.data.humancellatlas.org/",
+  "WrapperSs2RsemSingleSample.method": "Ss2RsemSingleSample",
+  "WrapperSs2RsemSingleSample.schema_version": "v3",
+  "WrapperSs2RsemSingleSample.run_type": "run",
+  "WrapperSs2RsemSingleSample.retry_seconds": 10,
+  "WrapperSs2RsemSingleSample.timeout_seconds": 120
+}
```

Comment:
I'm wondering why this looks much shorter comparing to `https://github.com/HumanCellAtlas/skylab/blob/master/hca-dcp/submit_example.json`

Reply:
fixed